src/baseclasses/chemical_energy/nome_cp_analysis.py

```diff
@@ -0,0 +1,145 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+
+from nomad.metainfo import Quantity, Reference, Section, SubSection, Datetime
+from nomad.datamodel.metainfo.basesections import Analysis, SectionReference, AnalysisResult, CompositeSystemReference
+from nomad.datamodel.results import Results, Material
+
+from baseclasses.chemical_energy import Chronopotentiometry
+from ..helper.utilities import get_reference
+
+
+class CPOERReference(SectionReference):
+    reference = Quantity(
+        type=Reference(Chronopotentiometry.m_def),
+        a_eln=dict(component='ReferenceEditQuantity', label='OER Chronopotentiometry Measurement'))
+
+
+class CPOERAnalysisResult(AnalysisResult):
```

Comment:
did you think of including the OCP voltage for the voltage shift?
and the resistance from EIS?

Further does it make sense to store a link to the setup and environment and sample in the analysis?

Reply:
also did you talk to marcel about activity meausre from LSV or galvonostatic sweep?

Reply:
Yes, voltage shift and R are actually things Marcel would also like to have in the explore view as histograms. But to do that I need to refresh my chemistry first and understand what EIS is.

Reply:
We did not talk about LSV so far.

Reply:
ok for EIS you ideally talk to frederik he has already code based on nomad queris for this

Reply:
lsv is the measurement to measure activity, this calculation here is a measure for stability